src/schema/rules/common_principles.yaml

```diff
@@ -0,0 +1,18 @@
+---
+# This file simply defines the order in which the Common Principles are presented in the specification.
+# The actual term definitions appear in the `objects/common_principles.yaml` file.
+- dataset
+- subject
+- session
+- sample
+- data_acquisition
+- data_type
+- task
+- event
+- run
+- modality
```

Comment:
```suggestion
- dataset
- modality
- data_type
- subject
- session
- sample
- data_acquisition
- task
- event
- run
```

Reply:
Trying to order this from "general" to "specific", I would bump modality and data_type higher up.

Not a hill I will die on.

Reply:
I was just replicating the existing organization, but I do agree that the principles could be reordered to be more informative. General to specific seems like the best way to go! 👍